requirements/insights-content-service.txt

```diff
@@ -0,0 +1,7 @@
+behave
+pytest
+psycopg2
+jsonschema
+requests
+minio
```

Comment:
Please remove from this file the requirements that are not related to these tests (minio, psycopg2, etc.) and install both `requirements/requirements.txt` and `requirements/insights-content-service.txt` in the runner script.

Reply:
if I remove for example `minio` I get:

`Exception ModuleNotFoundError: No module named 'minio'
Traceback (most recent call last):

.......

File "/home/dpensier/python/github.com/dpensi/insights-behavioral-spec/features/src/minio.py", line 17, in <module>
    from minio import Minio
ModuleNotFoundError: No module named 'minio'
`